### DISCUSSION TITLE
End-of-combo hitbursts (hit100k, hit300g, hit300k, etc.) are missing from Lazer

### DISCUSSION THREAD

**Discussion by nagi-desuuu** [ppy/osu#22405] [Q&A] [answered]
This is how it looks on lazer right now:

https://user-images.githubusercontent.com/75596775/214502731-7950e99d-c687-41c2-8c30-5859de406f60.mp4

And this is how it should look, like on stable:

https://user-images.githubusercontent.com/75596775/214503129-7f9c4c96-0117-4de9-af33-10e43dba7eb5.mp4

Beatmap: https://osu.ppy.sh/beatmapsets/974356#osu/2084725
Skin: https://osu.ppy.sh/community/forums/topics/190357?n=1

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/4625
